std/array.d

```diff
@@ -332,6 +335,165 @@ unittest
 }
 
 /**
+ * Interprets an array literal as a static array.
+ *
+ * Params:
+ *      arr = An array literal.
+ *
+ * Returns: A static array of size `arr.length`.
+ *
+ * Warning:
+ * Do not initialize a dynamic array with a call to staticArray.
+ * The dynamic array slice would point to stack memory no longer in use.
+ * Instead define the static array first using type inference (or `int[4]`).
+ * ---
+ * int[] invalid = [1,2,3,4].staticArray; // Wrong
+ * ---
+ * See $(LINK2 http://issues.dlang.org/show_bug.cgi?id=12625, Issue 12625).
+ */
+@nogc T[n] staticArray(T, size_t n)(T[n] arr)
+{
+    pragma(inline, true);
```

Comment:
Ok, you're the second person to say that so I guess I'll switch it.
But the [docs](http://dlang.org/spec/pragma.html#inline) and all but two existing uses in phobos use the `pragma(inline, true);` inside the function body. Try `ack 'pragma\(inline'`. Not to say that means its right.